ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.12.yaml

```diff
@@ -86,12 +86,11 @@ tests:
   steps:
     cluster_profile: vsphere-multizone
     workflow: cucushift-installer-rehearse-vsphere-ipi-zones-techpreview
-- as: installer-rehearse-nutanix
-  cluster: build01
+- as: installer-rehearse-ibmcloud
```

Comment:
pls do not touch with the existing test item, you need to add a dedicated test item for ibmcloud.